CONTRIBUTING.md

```diff
@@ -43,7 +43,8 @@ Those are common examples, for more information don't hesitate to have a look at
 When contributing, please keep in mind the following rules:
 
 - Make only non-breaking changes in minor versions. Enhancements to existing code are possible - please discuss it beforehand with the Otter team via a [feature request](#feature-requests).
-- If the new feature you are adding is replacing an existing one, please deprecate the old code in minor versions. Add the @deprecated tag in the JSDoc while mentioning the major version when it will be removed. Note that only **even** major Otter versions allow breaking changes. This means we only allow breaking changes in `n+2` major versions.
+- If the new feature you are adding is replacing an existing one, please deprecate the old code in minor versions. Add the `@deprecated` tag in the *JSDoc* while mentioning the major version when it will be removed. Note that only **even** major Otter versions allow important breaking changes (see [Versioning rules](./SECURITY.md)).\
```

Comment:
What's `important` means ?
we should give them a list of conditions to check if it is not important

Reply:
Need to be discussed indeed